chainerx_cc/chainerx/stack_vector.h

```diff
@@ -194,4 +195,33 @@ class StackVector {
     BaseContainer d_{};
 };
 
+// Formatter to print StackVector containing integral elements as e.g. '[]' or '[1, 2, 3]'.
+template <typename T, stack_vector_detail::size_type N>
+class DimArgFormatter {
+public:
+    static_assert(std::is_integral<T>::value, "DimArgFormatter requires intergral elements.");
+
+    explicit DimArgFormatter(const StackVector<T, N>& stack_vector) : stack_vector_{stack_vector} {}
+
+    void Print(std::ostream& os) const {
```

Comment:
How about `Format` instead of `Print`?
It matches the class name.

Reply:
Hm, I agree `Format` better matches the class name but I think `Print` is appropriate. The formatting is already implied by the class name while what's important to convey is that it also prints (of course this is also clear from the argument but still more important that reemphasizing the formatting). Actually, these "naming conventions" were borrowed from `array_repr.h`.